libs/build-nss-desktop.sh

```diff
@@ -57,12 +57,8 @@ fi
 # We do not know how to cross compile these, so we pull pre-built versions from NSS CI
 # https://github.com/mozilla/application-services/issues/962
 if [[ "${CROSS_COMPILE_TARGET}" =~ "darwin" ]]; then
-  #From https://firefox-ci-tc.services.mozilla.com/tasks/index/app-services.cache.level-3.content.v1.nss-artifact/latest
-  curl -sfSL --retry 5 --retry-delay 10 -O "https://firefox-ci-tc.services.mozilla.com/api/index/v1/task/app-services.cache.level-3.content.v1.nss-artifact.latest/artifacts/public/dist.tar.bz2"
-  SHA256="15e8cab63a4e0484564231a24d23d8be349217ba6add53b56609c339bb306829"
-  echo "${SHA256}  dist.tar.bz2" | shasum -a 256 -c - || exit 2
-  tar xvjf dist.tar.bz2 && rm -rf dist.tar.bz2
-  NSS_DIST_DIR=$(abspath "dist")
+  # From https://firefox-ci-tc.services.mozilla.com/tasks/index/app-services.cache.level-3.content.v1.nss-artifact/latest
+  NSS_DIST_DIR="$MOZ_FETCHES_DIR/dist"
```

Comment:
This also gets run outside of CI by devs who want to build locally.  Could this be inside an if branch?  I think there's an env variable named `CI` that could be use, if not `MOZ_FETCHES_DIR` seems like a fine test.